packages/flutter/lib/src/material/page.dart

```diff
@@ -267,12 +98,12 @@ class MaterialPageRoute<T> extends PageRoute<T> {
   Widget buildTransitions(BuildContext context, Animation<double> animation, Animation<double> forwardAnimation, Widget child) {
     if (Theme.of(context).platform == TargetPlatform.iOS &&
         Navigator.of(context).userGestureInProgress) {
-      return new _CupertinoPageTransition(
+      return new CupertinoPageTransition(
         animation: new AnimationMean(left: animation, right: forwardAnimation),
         child: child
       );
     } else {
-      return new _MountainViewPageTransition(
+      return new MountainViewPageTransition(
         animation: animation,
         child: child
```

Comment:
We probably want to do the same thing for MountainViewPageTransition.  (You can factor the common subexpression in to a variable, of course.)